Panels/FlexTabber.js

```diff
@@ -349,6 +351,7 @@ define([
                     }
                 }
                 frame._history_tabId.push(tabInfo.tabId);
+                Msg.broadcast("FlexTabActivated", tabInfo.tabId);
```

Comment:
Maybe also best to check whether no effects (that take time) are performed in the above code to activate the tab?

Reply:
Good point again :). The updateTabStates has several fade animation, both for the tab and for the header title. For how I use these messages it is not relevant that these animations aren't finished before broadcasting the message. I therefore left it as it is. 

But you are absolutely right that this is often important, especially if you need to measure view elements you want all animations to be finished. Usually, many lifecycle events are triggered, e.g. when transitions start and when they are finished (including all animations). But is difficult to add something like that in the existing code.